src/bigwig/index.ts

```diff
@@ -1,2 +1,3 @@
 export { HeaderData, FileType, CommonHeader, ZoomLevelHeader, BWTotalSummary, ChromTree } from "./BigWigHeaderReader";
+export { BigBedDataNarrowPeak, BigBedDataBroadPeak, BigBedDataMethyl, BigBedDataTssPeak } from "./encodeBigBed";
 export { BigWigData, BigBedData, BigZoomData, BigWigReader } from "./BigWigReader";
```

Comment:
Can you also make sure the parse functions are exported?

Reply:
And the `ParseFunction` type.

Reply:
Done.